subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -19,9 +19,9 @@ This chapter provides the information you need to migrate your older Gradle 4.x
 
 TIP: If you are using Gradle for Android, you need to move to version 3.4 or higher of both the Android Gradle Plugin and Android Studio.
 
-We recommend the following steps for all users:
+== For all users
 
- . If you are not already on version 4.10.3, read the sections <<#changes_5.0,below>> for help upgrading your project to 4.10.3.
+ . If you are not already on version 4.10.2+, read the sections <<#changes_4.10,below>> for help upgrading your project to 4.10.X. We recommend upgrading to version 4.10.2 or higher first because 4.10.2 is the minimum version that includes the Deprecations tab in the build scan plugin.
```

Comment:
Can we recommend 4.10.3 instead of 4.10.2 everywhere, please? We really mean _latest patch version of 4.x.x_. It's not just about `--scan` but to capture the maximum number of deprecations and provide the closest environment to 5.0 without actually making the jump.

Reply:
Only patch releases and probably not even that. So "latest 4.10.x release" is safe and legit.